thriftbp/prometheus.go

```diff
@@ -88,3 +88,93 @@ var (
 		Help: "The number of in-flight requests",
 	}, clientActiveRequestsLabels)
 )
+
+var (
+	serverConnectionsGauge = promauto.NewGauge(prometheus.GaugeOpts{
+		Name: "thrift_server_connections",
+		Help: "The number of client connections established to the service",
+	})
+)
+
+var (
+	ttlClientReplaceLabels = []string{
+		serverSlugLabel,
+		successLabel,
+	}
+
+	ttlClientReplaceCounter = promauto.NewCounterVec(prometheus.CounterOpts{
+		Name: "thrift_ttl_client_connection_houskeeping_total",
+		Help: "Total connection housekeeping done in thrift ttlClient",
+	}, ttlClientReplaceLabels)
+)
+
+const (
+	protoLabel = "proto"
+)
+
+var (
+	payloadSizeLabels = []string{
+		methodLabel,
+		protoLabel,
+	}
+
+	// 8 bytes to 8 megabytes
+	// some endpoints can have very small, almost zero payloads (for example,
+	// is_healthy), but we do have some endpoints with very large payloads
+	// (up to ~400KB).
+	payloadSizeBuckets = prometheus.ExponentialBuckets(8, 2, 20)
+
+	payloadSizeRequestHisto = promauto.NewHistogramVec(prometheus.HistogramOpts{
```

Comment:
`s/Histo/Bytes/`

It'll be clear in context that it is a histogram since you add a sample